### PR TITLE
Fix/blog code wrap

### PR DESCRIPTION
**Pull request by MildTomato**
— fix `<code/>` wrap issue on blog
— update RSS feed while were here :D

### CODE REVIEW

File: apps/www/public/rss.xml [hunk 1]
```diff
@@ -5,9 +5,16 @@
...
+      <lastBuildDate>Sun, 02 Oct 2022 16:04:00 +0000</lastBuildDate>
       <atom:link href="https://supabase.com/rss.xml" rel="self" type="application/rss+xml"/>
       <item>
+  <guid>https://supabase.com/blog/postgres-wasm</guid>
+  <title>Postgres WASM by Snaplet and Supabase</title>
+  <link>https://supabase.com/blog/postgres-wasm</link>
+  <description>We're open sourcing postgres-wasm, a PostgresQL server that runs inside a browser, with our friends at Snaplet.</description>
+  <pubDate>Sun, 02 Oct 2022 16:04:00 +0000</pubDate>
+</item>
+<item>
```
Comment (alaister) [inline apps/www/public/rss.xml:17]: Not sure if it's GitHub or not, but is the indentation strange here?

**Review by alaister — APPROVED** [review]
If the indentation is okay, then lgtm!

File: apps/www/public/rss.xml (hunk 1 above)
Comment (MildTomato) [reply apps/www/public/rss.xml:17]: it's auto generated file

Comment (MildTomato) [reply apps/www/public/rss.xml:17]: it should be indented though yea, maybe should run prettier afterwards

**Pull request merged**